policy/cnspec_policy.proto

```diff
@@ -147,6 +147,41 @@ message Bundle {
   repeated Framework frameworks = 8;
   repeated FrameworkMap framework_maps = 9;
   PolicyDocs docs = 5;
+  // Migrations for the current bundle, describes changes to its contents
+  // from older versions or other collections.
+  // Experimental as of v11.x
+  repeated MigrationGroup migration_groups = 11;
+}
+
+// MigrationGroup contains a set of migrations for a policy collection.
+// The policy should generally contain the UID and affected version.
+message MigrationGroup {
+  Policy policy = 1; // only informational fields will be used (version)
+  repeated Migration migrations = 2;
```

Comment:
is the intend here to descrieb migrations within the policy, i.e. queries being removed?

Reply:
Yes, and the 3 actions are: removal, creation, and modification (as seen in the enum)